wodles/aws/aws_s3.py

```diff
@@ -69,6 +70,12 @@
 
 # Enable/disable debug mode
 debug_level = 0
+INVALID_CREDENTIALS_ERROR_CODE = "SignatureDoesNotMatch"
+INVALID_REQUEST_TIME_ERROR = "RequestTimeTooSkewed"
```

Comment:
Naming conventions must be consistent. Use the `_ERROR` or `ERROR_CODE` suffix but stick to one of them.